core/src/main/java/org/infinispan/commands/module/ModuleCommandInitializer.java

```diff
@@ -11,9 +11,12 @@
  *
  * @author Manik Surtani
  * @since 5.0
+ * @deprecated Since 10.0, Commands which require initialization should implement {@link
```

Comment:
@ryanemerson my eagle eyes spotted this. ModuleCommandInitializer should been deprecated after introduction of InitializableCommand